src/sonic-yang-models/yang-models/sonic-bgp-monitor.yang

```diff
@@ -0,0 +1,90 @@
+module sonic-bgp-monitor {
+    namespace "http://github.com/Azure/sonic-bgp-monitor";
+    prefix bgpmon;
+    yang-version 1.1;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BGP Monitor";
+
+    revision 2022-01-11 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-bgp-monitor {
+        container BGP_MONITORS {
```

Comment:
This table fields seem to be duplicate from the table BGP_NEIGHBOR, please clarify why do we need a separate table, also, please use the grouping to group the fields and use them in multiple places.

Reply:
> This table fields seem to be duplicate from the table BGP_NEIGHBOR, please clarify why do we need a separate table

This is a good question, and I believe it is large than the PR trying to solve, ie, add the missing yang model for BGP_MONITORS table, which is existing for long,.

This PR will help minigraph parser unit test improvement, which validate each minigraph generated ConfigDB by SONiC Yang model.

This PR will also unblock the generic config updater end-to-end test.

We can create an issue that BGP_NEIGHBOR and BGP_MONITORS tables should be unified.

Reply:
Sure.

Reply:
Created issue https://github.com/Azure/sonic-buildimage/issues/9793